dashboard/src/components/localhost/AttendeeRequestList.vue

```diff
@@ -304,14 +294,14 @@ const rejectRequest = (member) => {
   changeLocalhostRequestStatus(member.name, 'Rejected').fetch()
 }
 
-const filterListByStatus = (filter) => {
+watch(selectedListFilter, (newFilter) => {
   requestByGroup.update({
     params: {
       hackathon: props.localhost.data.parent_hackathon,
       localhost: props.localhost.data.name,
-      status: filter.value,
+      status: FILTERS[newFilter].filters,
     },
   })
   requestByGroup.fetch()
-}
```

Comment:
Can you test and check the behavior on Firefox as well

Reply:
Yes did it too.